src/Microsoft.TemplateEngine.Cli/HelpAndUsage/TemplateInformationCoordinator.cs

```diff
@@ -351,7 +351,9 @@ private static void DisplayHintForOtherLanguages(INewCommandInput commandInput,
 
             string? preferredLanguage =
                 resolutionResult.UnambiguousTemplateGroupMatchInfo
-                    .TemplatesWithMatchingParametersForPreferredLanguage.First().GetLanguage();
+                    .TemplatesWithMatchingParametersForPreferredLanguage
```

Comment:
this is not needed. `TemplatesWithMatchingParametersForPreferredLanguage` should have same language